back/spec/services/sanitization_service_spec.rb

```diff
@@ -163,14 +163,6 @@
       expect(service.sanitize(input, features)).to eq input
     end
 
-    it 'allows vimeo iframe to pass through when video feature is enabled' do
```

Comment:
I removed this test because this is no-longer correct and the correct format is handled above